finetune/comparison_regressor.py

```diff
@@ -0,0 +1,107 @@
+import tensorflow as tf
+import numpy as np
+
+from finetune.target_encoders import RegressionEncoder
+from finetune.encoding import ArrayEncodedOutput
+from finetune.input_pipeline import BasePipeline
+from finetune.network_modules import regressor
+from finetune.base import BaseModel
+
+class ComparisonRegressionPipeline(BasePipeline):
```

Comment:
Can we factor out a comparison Pipeline that contains _text_to_ids,  _format_for_encoding and feed_shape_type_def and then have Comparison[Classification] and ComparisonRegression inherit from this?

Reply:
:+1: